Microsoft.Toolkit.Uwp.SampleApp/SamplePages/WeatherLiveTileAndToast/WeatherLiveTileAndToastCode.bind

```diff
@@ -1,40 +1,10 @@
 private void PopToast()
```

Comment:
Just putting this here, not sure what's wrong but noticed the 'Small' Tile example was blank?

![image](https://user-images.githubusercontent.com/24302614/97049952-665ec600-1531-11eb-8334-525f59b7fbc2.png)